forms.py

```diff
@@ -178,3 +178,22 @@ class Meta:
                 attrs={'summernote': summernote_attrs},
             ),
         }
+
+
+class GalleyForm(forms.ModelForm):
+    file = forms.FileField()
+
+    class Meta:
+        model = core_models.Galley
+        fields = (
+            'label',
+            'public',
+        )
+
+    def __init__(self, *args, **kwargs):
+        include_file = kwargs.pop('include_file', True)
+        super().__init__(*args, **kwargs)
+        self.fields['label'].required = False
+
+        if not include_file:
+            self.fields.pop('file')
```

Comment:
why do we need to re-declare the form here instead of importing from production?

Reply:
Future proofing I suppose - if we intend to make Typesetting part of the core we will do away with production.